src/app/modules/consultations/consultation-list/consultation-list.graphql.ts

```diff
@@ -1,4 +1,18 @@
 import gql from 'graphql-tag';
+import { environment } from '../../../../environments/environment';
+//TODO: Profanity filter feature, remove condition when ready fo deployment to production
+const fragments = {
+  locationId: !environment.production ? gql `
```

Comment:
Directly assign the query to the variable instead of storing it in the dictionary. Currently the key name is inaccurate and might lead to confusion as it is storing more than just location ID

Reply:
Done